.github/workflows/ci.yml

```
@@ -46,6 +46,10 @@ jobs:
             phpunit-version: 9
             symfony-version: '^6'
             stability: 'dev'
+          - php-version: '8.3'
+            phpunit-version: 9
+            symfony-version: '^7@dev'
```

once symfony 7 is released, remove the `@dev` before merging.
```suggestion
            symfony-version: '^7'
```

---

What is the expected release date?

---

november 2023.

i wanted to check with a project what the upcoming release will do, and had the conflict with the rmt so i thought i just check that as well and use that branch for the other project...